tools/maintenance/maintenance.py

```diff
@@ -45,10 +56,7 @@ def check_gcloud_components() -> None:
         raise LookupError(err_msg)
 
 def get_maintenance_nodes(project: str, regex: re = None) -> List[str]:
-    cmd = f"gcloud alpha compute instances list --project={project}" \
-            " --filter=scheduling.maintenanceInterval:PERIODIC " \
-            " --format='value(name)'"
-
+    cmd = PER_MAINT_CMD.format(project)
```

Comment:
I guess it's out of scope, but I would propose to not parse `gcloud`  output, instead use GCP Python SDK.
To address the concern of "extra dependencies", we could run the script in `gcloud`s virtualenv:
```
$ gcloud info --format="csv[no-heading](basic.python_location)"
> /usr/lib/google-cloud-sdk/platform/bundledpythonunix/bin/python3
```

Reply:
I will look into this in the future.  Do you know if it's always installed on all VM images?